src/EditDisplayPlistItem.swift

```diff
@@ -9,14 +9,15 @@
 import Cocoa
 
 @objc class EditDisplayPlistItem : NSMenuItem {
-	@objc var vendorID : UInt32
-	@objc var productID : UInt32
+	@objc var vendorID	  : UInt32
```

Comment:
Are these spaces intended?

Reply:
Oh no... the spaces are tabs, and this would be aligned when translate tabs as four spaces. I'll change all internal blanks to spaces soon.

Reply:
Fixed by commit jnooree/RDM@1cc0c78166aa879ee5a3d3be8bc32da69e6c4ecf